### PR TITLE
Add SubmitTransaction API page

### PR DESCRIPTION
**Pull request by petitnau**
Work on #2.
immagine
immagine
immagine

### CODE REVIEW

File: docs/api/algorand/submit-transaction.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# SubmitTransaction
+
+Send funds from a [DelegatedWallet](#TODO), to another (regular or delegated) wallet.
```
Comment (nikitaeverywhere) [inline docs/api/algorand/submit-transaction.mdx]: Please clarify whether the regular wallet will be opted in, linking `/docs/concepts/tokens#token-opt-in`

File: docs/api/algorand/submit-transaction.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+| `unsigned`  | `string[]`                        | The array of transactions (previously returned in the `noSign` array) that have not been signed. |
+| `signed`    | `string[]`                        | The array of transactions (previously returned in the `toSign` array) that have been signed by the user. |
+
+## Responses
+
+### Regular response
+
+|     Name    |         Type        | Description |
+| ----------- | ------------------- | ----------- | 
+| `txIds`     | `string[]`          | The array of transaction Ids of the transactions that have been submitted. |
```
Comment (nikitaeverywhere) [inline docs/api/algorand/submit-transaction.mdx]: No need to capitalize id

File: docs/api/algorand/submit-transaction.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+Returned when the parameters are invalid.
+
+|     Name    |                              Content                              |
+| ----------- | ----------------------------------------------------------------- |
+| Status code | 400                                                               |
+| `errorId`   | `INVALID_PARAMS`                                                  |
+| `message`   | `Validation errors in SubmitTransaction request.`                 | 
+| `errors`    | Array of strings containing additional informations.              |
+
+### Not issued by provider
```
Comment (nikitaeverywhere) [inline docs/api/algorand/submit-transaction.mdx:31]: Should it be `####` header?

**Review by nikitaeverywhere — APPROVED** [review]

File: docs/api/algorand/submit-transaction.mdx (hunk 1 above)
Comment (petitnau) [reply docs/api/algorand/submit-transaction.mdx]: Ah no sorry, this is a refuse from another page. Forgot to add a proper description

File: docs/api/algorand/submit-transaction.mdx (hunk 3 above)
Comment (petitnau) [reply docs/api/algorand/submit-transaction.mdx:31]: Not quite, these are all different kinds of errors (they are not a subcategory of validation errors). However, if you think that it is not clear, I can find some alternative.

**Comment by petitnau** [comment]
Wrongly pushed some other files, fixing.
Edit: done.

**Review by nikitaeverywhere — APPROVED** [review]

**Pull request merged**